src/daily-auths-report.tsx

```diff
@@ -258,6 +258,30 @@ function DailyAuthsReport(): VNode {
 
   return (
     <div>
+      <h2>Daily Auths Report</h2>
+      <div className="usa-accordion usa-accordion--bordered margin-bottom-2">
```

Comment:
Something we'll want to keep an eye out for: Some USWDS components have "initialization" logic, which only runs when `component.on()` is called. For the accordion, all the initialization does is [handle if `aria-expanded="true"`](https://github.com/uswds/uswds/blob/e1ed21783b089a2831504971b3426fbea1b5b752/src/js/components/accordion.js#L86-L91). We're not using that (currently), so it doesn't affect us. Since the primary toggling behavior is handled by event delegation, it works fine as-is.

If we'd need to run initialization behavior, it'd be something where I expect we'd have to do something like this instead:

```js
function Accordion() {
  const accordionRef = useRef();
  useEffect(() => {
    accordion.on(accordionRef.current?);
  }, []);

  return <div ref={accordionRef} className="usa-accordion" />;
}
```

This varies by component as well, since other components rely heavily on initialization.

(Another thing I'm [keen to improve](https://github.com/uswds/uswds/issues/2884#issuecomment-872238628))